.gitignore

```diff
@@ -84,3 +84,5 @@ terraform.rc
 
 .eslintcache
 /clients/java/.openapi-generator/FILES
+docs/Gemfile
```

Comment:
Why?  AFAIK this file is actively used; we should not make it ignored-by-default on Git.

(We can find other ways to help avoid changing it, let's talk!)

Reply:
Removed. Let's discuss it! happy to learn:)